src/qibolab/_core/instruments/qm/config/config.py

```diff
@@ -79,48 +86,92 @@ def configure_dc_line(
     ):
         controller = self.controllers[channel.device]
         if controller.type == "opx1":
-            controller.analog_outputs[channel.port] = AnalogOutput.from_config(config)
+            keys = ["offset", "filter"]
         else:
-            controller.analog_outputs[channel.port] = FemAnalogOutput.from_config(
-                config
-            )
+            keys = list(config.model_fields.keys())
```

Comment:
Why has this been changed, if the `.from_config()` method is still around?

Reply:
Following also your other comment below, I believe `FemAnalogOutput` no longer exists in this branch (it was dropped together with `AnalogOutput`).